content/v8.0.0/index.yml

```diff
@@ -0,0 +1,34 @@
+layout: event
+
+header: /_shared/header.md
+
+type: event
+pagetitle: Design System meetup Sydney v8.0.0
+version: v8.0.0
+city: Sydney
+date: 2019-05-13
+location: /_locations/hipages.md
+sponsors:
+  - /_sponsors/canva.md
+  - /_sponsors/thinkmill.md
+  - /_sponsors/tomwalker.md
+link: https://www.meetup.com/Design-System/events/258516904
+eventdescription: |
+  Design System meetup Sydney v8.0.0 is up.
```

Comment:
minor - but shouldn't the event description for events that have already passed just be `Design System meetup Sydney v8.0.0`? is up seems strange for events that are already past.

LMK if you agree, and I can separately fix it up for all events in another PR.

Reply:
meh... it's for better infos later... just haven't come up with them